Subject: [PATCH] READ_PHONE_STATE permission for IMS

external/ims would grant this to this shared UID, but we don't
 build those packages

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 1c37a0a..1a92ba5 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -152,6 +152,8 @@
     <uses-permission android:name="android.permission.READ_BLOCKED_NUMBERS" />
     <!-- Needed for emergency contact notification. -->
     <uses-permission android:name="android.permission.WRITE_BLOCKED_NUMBERS" />
+    <!-- Qcom IMS Permission -->
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
 
     <!-- This tells the activity manager to not delay any of our activity
          start requests, even if they happen immediately after the user
